docs/apis-tools/tasklist-api-rest/tasklist-api-rest-authentication.md

```diff
@@ -19,7 +19,13 @@ For example, send a request using _curl_:
 curl -X POST -H -H :accept: application/json" -H "Authorization: Bearer <TOKEN>" -d '' http://localhost:8080/v1/tasks/search
 ```
 
-### How to obtain the access token
+Or, with SaaS:
+
+```shell
+curl -X POST --header 'content-type: application/json' --data '{"client_id": "<client-id>", "client_secret":"<client-secret>","audience":"<audience>","grant_type":"client_credentials"}' https://<authorization server url>
```

Comment:
or are you looking for something like https://login.cloud.camunda.io/oauth/token

Reply:
OR the audience, like zeebe.camunda.io